web/src/games/cardtable/CribbageBoardB.tsx

```diff
@@ -0,0 +1,222 @@
+import React, { FunctionComponent } from 'react';
+import css from './CribbageBoardB.css';
+import { IScore, IScoreKeeper } from './game';
+import Dialog from '@material-ui/core/Dialog';
+import Slider from '@material-ui/core/Slider';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import Button from '@material-ui/core/Button';
+
+interface PegholeProps {
+  idx: number;
+  className: string;
+  score: IScore;
+  opponent: boolean;
+}
+
+function Peghole(props: PegholeProps) {
+  let clazz = css.Peghole;
+
+  const isPegged = (idx: number) => {
+    let pegged = false;
```

Comment:
You can replace isPegged implementation with:

``return  (idx < 1 && props.score.back === idx)  ||
               (idx < 1 && props.score.front === idx) ||
               (idx < 61 && props.score.front === idx) ||
               (idx < 61 && props.score.back === idx) ||
               (props.score.front > 60 && props.score.front % 60 === idx) ||
               (props.score.back > 60 && props.score.back % 60 === idx);

Reply:
I'll take a look, some of the logic was pretty twisty, I'll review and correct.